src/links.scss

```diff
@@ -723,6 +723,52 @@ $color-white: #fff;
 
 }
 
+/*
+ * highlight13
+ */
+
+ @import url(https://fonts.googleapis.com/css?family=Patrick+Hand+SC);
+
+ .highlight13 {
+   --style-color1: #000000;
+   --style-color2: #fdf3a7;
+   --style-color3: #bcf2fd;
+   --style-color4: #c0fcab;
+   --style-color5: #f6c9c8;
+   --style-color6: #b9cafb;
+   --style-color7: #eeeeee;
+   
+   text-decoration: none;
+   font-family: "Patrick Hand SC", cursive;
+   padding: $padding;
+   transition: all 0.5s ease;
+   color: var(--style-color1);
+   letter-spacing: 1px;
+   outline: none;
+   box-shadow: 20px 38px 34px -26px hsla(0, 0%, 0%, 0.2);
+   border-radius: 255px 15px 225px 15px/15px 225px 15px 255px;
+   &:hover {
+     box-shadow: 2px 8px 4px -6px hsla(0, 0%, 0%, 0.5);
+   }
+   &.yellow {
```

Comment:
If you check out the **highlight10**, it is similar. if you change the background color then it will be the same as the one you submitted.

so instead can you please add the background color properties to the highlight10. this way, your style will be added and it won't add duplicate style to the package.